### ISSUE TITLE
SearchRoot is not setup properly

### ISSUE THREAD

**Issue by dwmkerr** [dwmkerr/app-icon#50]
Run in a debugger, the usage of `searchRoot` in `generate` appears to be incorrect. Also appears to not accept arrays.

**Comment by timri** [comment]
This line https://github.com/dwmkerr/app-icon/blob/45df2e47cea1dfe0bcfdd6c9e6b0c9572aadfff1/bin/app-icon.js#L45
should instead be
```js
return generate({ sourceIcon: icon, searchRoot: search, platforms });
```

**Comment by dwmkerr** [comment]
Thanks @timri I will sort this one!

**Comment by punksta** [comment]
I am having very annoying issue. I use this command
```
node_modules/app-icon/bin/app-icon.js generate  --platforms=android --search ./android/app/src/main
```
but it's trying to find all manifest files in current dir. I have CI cache under this folder. 
So it's finding all manifests of every external lib and generating images for each of them. I think it is generating about 150 images.

![image](https://user-images.githubusercontent.com/13404754/52742116-1c95ac80-2fe8-11e9-93e8-98025258af9a.png)

![image](https://user-images.githubusercontent.com/13404754/52742145-29b29b80-2fe8-11e9-8768-ccf196840493.png)

**Comment by dwmkerr** [comment]
OK definitely looks like a bug! Let me set up something similar locally and see if I can reproduce

**Comment by dwmkerr** [comment]
Resolved, and released as `0.8.1`

**Comment by punksta** [comment]
[dwmkerr](/dwmkerr) cool!
